lib/bitcoin/README.md

```diff
@@ -66,14 +47,16 @@ Linux users:
 ifconfig
 ```
 
-If you installed Bcoin locally, `localhost` will do.
+If you installed Bitcoin locally, `localhost` will do.
 
-In the following configuration `bcoin.local` refers to your IP address, DNS address, or `localhost`.
+In the following configuration `bitcoin.local` refers to your IP address, DNS address, or `localhost` of the Bitcoin peer.
 
 Edit your bitcoin-config.json
 ```json
 {
-  "bitcoinPeerUri": "[FILL THIS IN!]",
+  "bitcoinPeerUri": "bitcoin.local:18332",

```

Comment:
Sanity: is this kubernetes cluster artifacts?
```suggestion
  "bitcoinPeerUri": "http://localhost:18332",
```

Reply:
No. Per the line above, I did not want to assume where your bitcoin peer is. "bitcoin.local" would never resolve unless you had explicitly set your network domain to "local" or wrote in your host file, which makes it a safe domain to use for examples.